tls/s2n_security_policies.c

```diff
@@ -27,6 +26,9 @@ const struct s2n_security_policy security_policy_20170210 = {
     .ecc_preferences = &s2n_ecc_preferences_20140601,
 };
 
+const struct s2n_security_policy *S2N_SECURITY_POLICY_DEFAULT = &security_policy_20170210;
```

Comment:
I think we need to export all of the available policies in order to deprecate the version functions. Otherwise customers are forced to continue to use deprecated APIs.

Reply:
After the discussion I've left the 4 policies exposed, and I have not deprecated the other functions. We want to expose a builder to make this policy api work for customers (#2380).